source/ui/Components/Output/LightstripSettingsWindow.qml

```diff
@@ -85,8 +85,17 @@ Window {
                 // Reads the current input and passes the new value to the model.
                 function processInput() {
                     var inputValue = controlsColumn.integerFromInputString(String(baseAddressTextField.text))
-                    if (inputValue !== undefined && inputValue > 0) {
+                    var maxSize = outputComponent.getMemorySize();
+                    if (inputValue !== undefined && inputValue >= 0 && inputValue <= maxSize) {
```

Comment:
I think that should be `inputValue < maxSize`. Entering 1024 currently leads to crash on my machine.